scripts/ddp-RxTerms-script/src/utils/common.ts

```diff
@@ -0,0 +1,12 @@
+export const parseToHOCON = require("hocon-parser");
+
+const fs = require("fs");
+
+export const writeFile = (fileName: string, value: string) => {
+  return fs.writeFile(fileName, value, (err: string) => {
+    if (err) {
+      return console.log(err);
+    }
+    console.log("The file was saved!");
+  });
+};
```

Comment:
Consider just getting rid of this file and move write file to index.ts . We are just calling fs.writefile. Up to you.

Reply:
Agree, removed